tests/simulator/user_modeling/test_preference_model.py

```diff
@@ -55,59 +56,45 @@ def test_initial_item_preferences_sip(preference_model_sip):
 
 
 def test_get_slotvalue_preference_sip(preference_model_sip):
-    # Given
     slot = "GENRE"
     value = "Comedy"
 
-    # When
     preference = preference_model_sip.get_slotvalue_preference(slot, value)
 
-    # Then
     assert -1 <= preference <= 1
 
 
 def test_get_slotvalue_preference_pkg(preference_model_pkg):
-    # Given
     slot = "GENRE"
     value = "Drama"
 
-    # When
     preference = preference_model_pkg.get_slotvalue_preference(slot, value)
 
-    # Then
     assert preference == 1
 
 
 def test_get_item_preference_sip(preference_model_sip):
-    # Given
     item_id = "10"
 
-    # When
     preference = preference_model_sip.get_item_preference(item_id)
 
-    # Then
     assert -1 <= preference <= 1
 
 
 def test_get_item_preference_pkg(preference_model_pkg):
-    # Given
     item_id = "100"  # geners = ['Drama', 'Thriller']
 
     # When (Both Drama and Thriller are favored as 1)
     preference = preference_model_pkg.get_item_preference(item_id)
 
-    # Then
     assert preference == 1
 
 
 def test_get_slot_preference_pkg(preference_model_pkg):
-    # Given
     slot = "GENRE"
 
-    # When
     slot, preference = preference_model_pkg.get_slot_preference(slot)
```

Comment:
just put the slot value inline and remove the line above

Reply:
Done.